cpp/merkletree/tree_hasher.h

```diff
@@ -10,28 +13,28 @@ class TreeHasher {
  public:
   // Takes ownership of the SerialHasher.
   TreeHasher(SerialHasher* hasher);
-  ~TreeHasher();
 
   size_t DigestSize() const {
     return hasher_->DigestSize();
   }
 
-  std::string HashEmpty();
+  const std::string& HashEmpty() const {
```

Comment:
I thought about it, but it causes a cascade of other changes, didn't really want to get into that for now...